django/utils/autoreload.py

```diff
@@ -285,6 +285,16 @@ def restart_with_reloader():
     while True:
         args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions] + sys.argv
         new_environ = os.environ.copy()
+        if _win and six.PY2:
+            # In some Python 2 cases, such as when using
+            # __futures__.unicode_literals, environment keys and values may
+            #  get unicode values.  Windows platforms need str objects. Here
+            #  we coerce both to the str type.
+            for key in new_environ.keys():
+                str_key = str(key)
+                str_value = str(new_environ[key])
```

Comment:
I guess this is going to crash if the value contains non-ASCII chars? Maybe we should be using `force_bytes(s, encoding=SYSTEM_ENCODING)` instead of a bare `str()` call? I'm not a Windows expert but I think we have similar handling of data the localization tools.